extension-manifest-v3/src/background/cosmetics.js

```diff
@@ -45,7 +45,7 @@ const adblockerStartupPromise = (async function () {
   );
 })();
 
-async function adblockerInjectStylesWebExtension(
+function adblockerInjectStylesWebExtension(
```

Comment:
it doesn't change anything, right? (it's fine to simplify, just changing for potential regressions)